modules/sdk/gradle-plugins-app-docker/src/gradleTest/smoke/build.gradle

```diff
@@ -73,7 +73,7 @@ runGradleTest {
 	onNext {
 		Image image ->
 
-		if (image.repoTags.contains("example/sample:foo")) {
+		if (image.repoTags != null && image.repoTags.contains("example/sample:foo")) {
```

Comment:
@Seiphon I'm not able to reproduce this issue locally. What is the cause of the issue?